integrations/applications/custom-application/resources/info.json

```diff
@@ -1,6 +1,7 @@
 {
     "id": "custom-application",
     "name": "Standard-Based Application",
+    "version": "v1.0.0",
```

Comment:
Do we need to use `v` prefix?

Reply:
https://github.com/wso2-extensions/identity-integration-ui-templates/pull/26/commits/f90cb5b4c956d1f4fc1442db737f7f89cbcc23a2